tests/test_domino.py

```diff
@@ -16,3 +28,19 @@ def test_versioning(requests_mock, dummy_hostname):
     with pytest.raises(Exception):
         dom.requires_at_least("5.11.0")
 
+def test_request_session():
+     request_manager = _HttpRequestManager(auth=TestAuth())
+     start_time = time.time()
+     try:
+         response = request_manager.request_session.get(
+             'https://httpbin.org/delay/10',
```

Comment:
Do we actually want to hit httpbin? Or should this. behavior be mocked?

Reply:
Updated.